examples/servers/reading-list/service.go

```diff
@@ -9,7 +9,7 @@ import (
 	"os"
 	"time"
 
-	"cloud.google.com/go/trace"
+	trace "cloud.google.com/go/trace/apiv2"
```

Comment:
Sorry but I tried compiling this and it fails with:

```
./service.go:35:32: cannot use pid (type string) as type option.ClientOption in argument to "cloud.google.com/go/trace/apiv2".NewClient:
	string does not implement option.ClientOption (missing Apply method)
./service.go:60:18: s.tracer.HTTPHandler undefined (type *"cloud.google.com/go/trace/apiv2".Client has no field or method HTTPHandler)
./service.go:110:18: s.tracer.GRPCServerInterceptor undefined (type *"cloud.google.com/go/trace/apiv2".Client has no field or method GRPCServerInterceptor)
```

Does it work for you? Am I using the right version of the trace paclage?

Reply:
I am having same problem with cloud.google.com/go/trace/apiv1. I did not compile main.go in /server before making this PR. I should have, sorry. I am not sure how this ever worked. The version in the mod.go file is the latest release 0.56.0: https://github.com/googleapis/google-cloud-go/releases. @jprobinson made this commit. Perhaps he can be of help.